src/main/scala/GameRunner.scala

```diff
@@ -0,0 +1,113 @@
+import GameHelpers._
+import GamePrompts._
+import scala.util.Random
+
+object GameRunner extends App {
+
+  def gameStart(): Unit = {
+    println()
+    println("Welcome to \u001b[31;1mScala Scrabble Practice")
+    println()
+    val name: String = promptName()
+    println()
+    println("Hello " + name + ". \u001b[0mThe Human plays against the Computer.")
+    println()
+    val level: String = promptLevel()
+
+    val letterBag = Bag.initialise()
+    val human = Player(name, Nil, Nil, false)
+    val computer = Player("The Computer", Nil, Nil, false)
+    val newGame = Game(letterBag, human, computer, level)
+
+    humanTurn(newGame)
```

Comment:
This logic of each `xTurn` calling the next one a bit tricky. How about a loop that checks whether the `Game` is over and has a pointer to whose turn is next? Then none of the `xTurn` methods need to know who else they are playing, they just take their turn when prompted and then wait for their go again. This would make it simpler to reason about as the `Player` model has fewer concerns.

In addition, would it be possible to model the distinction between a Human and Computer player directly? Then your game interface could be much simpler, such as 

```scala
def takeTurn(game: Game, player: Player): Game
```

If you can get to the point where you've abstracted away the differences between a Human and a Computer, you can write a single `takeTurn` method and everything will fall into place. This would then also allow you to only write some of the logic once e.g. calls to `checkGameOver`, `displayRack`, `word match etc.